### PR TITLE
Refactor/cleanup overrideDefaultMode

### PR DESCRIPTION
**Pull request by cdrini**
Refactor these functions to be easier to read/less verbose.

### CODE REVIEW

**Comment by cdrini** [comment]
@mheiman Would you be able to take a quick look at this one? Cleaning up some of the code around the area in your recent PR :)

**Comment by mheiman** [comment]
Looks good. You haven't changed any behaviors that I can see.

If you want to improve clarity, changing `isMobileBreakpoint` to something like `underMobileBreakpoint` or `withinMobileBreakpoint` would help.

File: src/BookReader.js [hunk 1]
```diff
@@ -452,60 +452,56 @@ BookReader.prototype.readQueryString = function() {
...
-    nextMode = params.mode;
-  } else if (ifMobileBreakpoint()) {
-    nextMode = this.constMode1up;
+  const windowWidth = $(window).width();
+  const isMobileBreakpoint = windowWidth && windowWidth <= this.onePageMinBreakpoint;
+
+  let initialMode;
+  if (params.mode) {
+    initialMode = params.mode;
+  } else if (isMobileBreakpoint) {
```
Comment (cdrini) [inline src/BookReader.js:465]:
```suggestion
  const isMobile = windowWidth && windowWidth <= this.onePageMinBreakpoint;

  let initialMode;
  if (params.mode) {
    initialMode = params.mode;
  } else if (isMobile) {
```

**Comment by cdrini** [comment]
Thanks @mheiman ! I changed it to `isMobile`.

**Pull request merged**